Spec/Auth.swift

```diff
@@ -1717,14 +1718,12 @@ class Auth : QuickSpec {
                     waitUntil(timeout: testTimeout) { done in
                         auth.authorise(nil, options: authOptions) { tokenDetails, error in
                             expect(authCallbackHasBeenInvoked).to(beTrue())
-
-                            authCallbackHasBeenInvoked = false
```

Comment:
Yes but if no arguments are passed in, then a TokenParams object has not been passed in.

What it actually states is:

>  TokenParams and AuthOptions are optional. When the arguments are present...

The point is that if you don't provide those arguments or they are null (but not empty TokenParam or AuthObtion objects), then the defaults are not overriden.